### DISCUSSION TITLE
NFS for Bulk storage. What should i exclude?

### DISCUSSION THREAD

**Discussion by Handkep** [nextcloud/all-in-one#5432] [Questions] [answered]
I have to use NFS for bulk storage and store currently everything in the data directory on a NFS Share in the local 1G Network. But i would like to speed up the server (current response time is 500-1400ms). I know that SQLite is a nogo over NFS. What files/folders do you recomend to store locally on a (rather small) SSD but keep the big userdata on an NFS Drive?

**Comment by szaimen** [comment, marked as answer]
Hi, in general I would recommend to only put the datadir onto the nfs drive. The rest should stay on SSD storage. See https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-change-the-default-location-of-nextclouds-datadir
